README.md

```diff
@@ -26,27 +26,37 @@ You can install it by following the [documentation](https://docs.oscar.grycap.ne
 
 ### 🧑‍💻 Setting Up the Configuration File
 
-The test suite uses environment variables to store sensitive information such as  endpoints and credentials.
+The test suite uses environment variables to store sensitive information such as endpoints and credentials. I'd recommend that you have two environment files. The first includes the cluster information, and the second contains the authentication process credentials. This way, you can switch between authentication processes such as EGI-CheckIn or Keycloak. Also, you can create one environment file that contains all the information.
 
 Create a `.env.yaml` file according to the template shown in `env-template.yaml`
 
-The following information is required:
-
+The following information is required about the cluster information:
   - `OSCAR_ENDPOINT`: The endpoint of the OSCAR cluster (e.g. https://mycluster.oscar.grycap.net) 
   - `OSCAR_METRICS`: The endpoint of the OSCAR metrics.
   - `OSCAR_DASHBOARD`: The endpoint of the OSCAR UI (dashboard).
   - `BASIC_USER:`: Base64-encoded information of the authentication for the 'oscar' user (echo -n "oscar:password"  | base64)
-  - `EGI_AAI_URL`: The base URL of the EGI AAI (Authentication and Authorisation Infrastructure) server.
-      - For the production server, use `https://aai.egi.eu`.
-      - For the demo server, use `https://aai-demo.egi.eu`.
-  - `REFRESH_TOKEN`: The OIDC token used to automate the execution of the test suite. In order to get a Refresh Token, head to the [Check-in Token Portal](https://aai.egi.eu/token/) or [Demo Check-in Token Portal](https://aai-demo.egi.eu/token/), click **Authorise** and then **Create Refresh Token** button to generate a new token.
```

Comment:
From the description it looks like refresh tokens can only be used with EGI Check-In. These should be possible to use also from Keycloak.